src/sentry/assistant/guides.py

```diff
@@ -110,4 +110,18 @@
             }
         ],
     },
+    "dynamic_counts": {
+        "id": 4,
```

Comment:
We most recently used `id:4` for the discover sidebar guide. You might end up with some overlap if you re-use that id.

Reply:
Should I use 5 instead?

Reply:
I took a look in production and 5 has some usage, but 7 is unused.